src/domains/fdr-app/api/fdr-fase1/nodoPerPa/v1/fdr_pagopa.xml.tpl

```diff
@@ -1,11 +1,13 @@
 <policies>
     <inbound>
         <base />
+        <set-variable name="enable_fdr_org_soap_request_switch" value="{{enable-fdr-org-soap-request-switch}}" />
+        <set-variable name="is_fdr_nodo_pagopa_enable" value="@(${is-fdr-nodo-pagopa-enable})" />
         <choose>
-            <when condition="@(${is-fdr-nodo-pagopa-enable})">
+            <when condition="@( context.Variables.GetValueOrDefault<bool>("is_fdr_nodo_pagopa_enable", false) && context.Variables.GetValueOrDefault<string>("enable_fdr_org_soap_request_switch", "").Equals("true") )">
```

Comment:
Why `GetValueOrDefault<string>` instead of `GetValueOrDefault<bool>`? I think `is_fdr_nodo_pagopa_enable` is also a named value so it is possible to cast directly in boolean value, correct me if I'm wrong.

Reply:
`${is-fdr-nodo-pagopa-enable}` is not a named value, while `enable_fdr_org_soap_request_switch` is a named value and is treated as a string